make_deb

```diff
@@ -0,0 +1,163 @@
+BUILD_ROOT="${GIT_ROOT}/packages/build"
+
+# Final package name
+package_name="foglamp-${plugin_type}-${plugin_name}-${plugin_version}-${architecture}"
+
+# Print the summary of findings
+echo "The package root directory is   : ${GIT_ROOT}"
+echo "The FogLAMP required version    : ${foglamp_version}"
+echo "The package will be built in    : ${BUILD_ROOT}"
+echo "The architecture is set as      : ${architecture}"
+echo "The package name is             : ${package_name}"
+echo
+
+# Create the package directory. If a directory with the same name exists,
+# it is copied with a version number.
+
+# First, create the BUILD_ROOT folder, if necessary
+if [ ! -L "${BUILD_ROOT}" -a ! -d "${BUILD_ROOT}" ]; then
+    mkdir -p "${BUILD_ROOT}"
+fi
+
+
+# Check if the default directory exists
+if [[ ! -d "${FOGLAMP_ROOT}" ]]; then
+    logger -p local0.err -t "foglamp.script.foglamp" "FogLAMP cannot be executed: ${FOGLAMP_ROOT} is not a valid directory."
+    echo "FogLAMP cannot be executed: ${FOGLAMP_ROOT} is not a valid directory."
+    echo "Create the enviroment variable FOGLAMP_ROOT before using FogLAMP."
+    echo "Specify the base directory for FogLAMP and set the variable with:"
+    echo "export FOGLAMP_ROOT=<basedir>"
+    exit 1
+fi
+
+# Check/set LD_LIBRARY_PATH
+libPathSet=0
+libdir=${FOGLAMP_ROOT}/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
+libdir=${FOGLAMP_ROOT}/cmake_build/C/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
+[ "$libPathSet" -eq "0" ] && echo "Unable to set/update LD_LIBRARY_PATH to include path of Foglamp shared libraries: check whether ${FOGLAMP_ROOT}/lib or ${FOGLAMP_ROOT}/cmake_build/C/lib exists" && exit 1
+
+rm -rf build; mkdir ./build; cd ./build; cmake ..; make; cd ..
+
+
+cd "${BUILD_ROOT}"
+existing_pkgs=`find . -maxdepth 1 -name "${package_name}.????" | wc -l`
+existing_pkgs=$((existing_pkgs+1))
+new_stored_pkg=$(printf "${package_name}.%04d" "${existing_pkgs}")
+if [ -d "${package_name}" ]; then
+    echo "Saving the old working environment as ${new_stored_pkg}"
+    mv "${package_name}" "${new_stored_pkg}"
+fi
+mkdir "${package_name}"
+
+# Populate the package directory with Debian files
+# First with files common to all pla
+echo -n "Populating the package and updating version file..."
+cd "${package_name}"
+cp -R ${GIT_ROOT}/packages/Debian/common/* .
+cp -R ${GIT_ROOT}/packages/Debian/${architecture}/* .
+sed -i "s/Version: 1.0.0/Version: ${plugin_version}/g" DEBIAN/control
+sed -i "s/Depends: foglamp/Depends: foglamp (${foglamp_version})/g" DEBIAN/control
+
+mkdir -p usr/local/foglamp
+cd usr/local/foglamp
+mkdir -p "plugins/${plugin_type}/${plugin_install_dirname}"
```

Comment:
It should be under `services`. Like below
```
mkdir -p "${plugin_type}"
cp -R --preserve=links ${GIT_ROOT}/build/foglamp.services.notification* "${plugin_type}"

```

Reply:
It's not a plugin, it is a service - I know we have these names because it is a copy/paste of the plugins make_deb script, but let's make it a real service please.